.vscode/launch.json

```diff
@@ -0,0 +1,61 @@
+{
+    "configurations": [
```

Comment:
Please remove

Reply:
why? how do you debug this project otherwise?